packages/rendermime/src/rendermime.ts

```diff
@@ -43,20 +36,32 @@ import {
  * render the model into a widget.
  */
 export
-class RenderMime implements IRenderMime {
+class RenderMime {
   /**
    * Construct a renderer.
    */
   constructor(options: RenderMime.IOptions = {}) {
-    if (options.items) {
```

Comment:
Wait, if we already had an `items` option in the constructor, why did we *also* have the `addDefaultRenderers` static method? That's what was confusing me. I think a default items option is fine.